terraform/teuthology/ecp/worker.tf

```diff
@@ -52,7 +52,7 @@ resource "openstack_compute_instance_v2" "node" {
     key_pair = "${var.os_keyname}"
     user_data = "${file("${path.module}/../../../openstack/userdata-ecp.yaml")}"
     security_groups = ["default", "sesci"]
-    metadata {
+    metadata = {
```

Comment:
This should be applied to other worker.tf files in repo too.

Reply:
Also this commit is not related to the topic of the PR, can you raise another PR including all related to terraform 12 changes?